src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1800,65 +1800,47 @@
   <data name="None" xml:space="preserve">
     <value>None</value>
   </data>
-  <data name="HighDpiOption_UseDPIOnLogin" xml:space="preserve">
-    <value>On Windows login</value>
+  <data name="CompatibilityOnWindowsLogon" xml:space="preserve">
+    <value>On Windows log on</value>
```

Comment:
`Login` and `log on` are both fine so to avoid the need to retranslate, I think we should keep it as "login".

Reply:
Both are fine, but if we change strings translators will need to translate it again.

Reply:
Ok

Reply:
Once we finalize the strings in English, we'll need to do a find and replace to update the strings in other languages.

Reply:
We did that, I know, but that actually didn't work. I found that the changed string became one of candidates in Crowdin.

Reply:
It should work but if not, we can manually sync the updated files.